rust/libnewsboat/src/matcher.rs

```diff
@@ -60,19 +60,25 @@ impl Operator {
                 }
 
                 Err(errmsg) => Err(MatcherError::InvalidRegex {
-                    regex: value.0.clone(),
+                    regex: value.literal().to_string(),
                     errmsg,
                 }),
             },
             Operator::NotRegexMatches => Operator::RegexMatches
                 .apply(attr, value)
                 .map(|result| !result),
-            Operator::LessThan => Ok(string_to_num(attr) < string_to_num(&value.0)),
-            Operator::GreaterThan => Ok(dbg!(string_to_num(attr)) > dbg!(string_to_num(&value.0))),
-            Operator::LessThanOrEquals => Ok(string_to_num(attr) <= string_to_num(&value.0)),
-            Operator::GreaterThanOrEquals => Ok(string_to_num(attr) >= string_to_num(&value.0)),
+            Operator::LessThan => Ok(string_to_num(attr) < string_to_num(&value.literal())),
+            Operator::GreaterThan => {
+                Ok(dbg!(string_to_num(attr)) > dbg!(string_to_num(&value.literal())))
```

Comment:
We certainly don't need them here, since they'll print stuff even in a release build. Thanks for catching that! I was editing the code with regexes, so didn't notice this :) I'll add a CI job that checks for them.